packages/avatar/src/react/index.js

```diff
@@ -9,10 +9,10 @@ import { getColorByName, getInitials, transformSrc } from '../js/index.js'
 
 const styles = {
   avatar: ({ size }) =>
-    css({
-      ...stylesheet['.psds-avatar'],
-      ...stylesheet[`.psds-avatar--size-${size}`]
-    }),
+    compose(
```

Comment:
Couldn't this also be written as `css(stylesheet['.psds-avatar'], stylesheet[.psds-avatar--size-${size}])` Is there an advantage to using compose? Should we be using compose all the time now?